src/res_scanner.ml

```diff
@@ -118,7 +118,7 @@ let scanIdentifier scanner =
   ) do
     next scanner
   done;
-  let str = Bytes.sub_string scanner.src startOff (scanner.offset - startOff) in
+  let str = (String.sub [@doesNotRaise]) scanner.src startOff (scanner.offset - startOff) in
```

Comment:
`Bytes.sub_string` also raised exception, but reanalyze doesn't know yet. Fixed in reanalyze master